Solutions/CiscoDuoSecurity/Analytic Rules/CiscoDuoADSyncFailed.yaml

```diff
@@ -0,0 +1,28 @@
+id: 398dd1cd-3251-49d8-b927-5b93bae4a094
+name: Cisco Duo - Admin user deleted
+description: |
+  'Detects when admin user is deleted.'

```

Comment:
Wondering if the name and description should be AD Sync Failed looking at the logic. It seems there is another analytic rule for Admin Delete.